src/extras-openmoji/climate-environment/E2D1.svg

```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All the blades should be evenly spaced around the center of the turbine

Reply:
![2020-02-27 at 12 13](https://user-images.githubusercontent.com/480224/75439679-8b774680-595a-11ea-8c9b-28cabbccfaf5.jpg)